src/Utility/Inflector.php

```diff
@@ -304,7 +301,6 @@ class Inflector
         'Ơ' => 'O',
         'Ø' => 'O',
         'Ǿ' => 'O',
-        'Ö' => 'Oe',
```

Comment:
The duplicate is here: https://github.com/cakephp/cakephp/pull/7558/files#diff-19eff47ebbcb312510b116f11192a975R176